src/Clap-Core.package/Main.class/properties.json

```diff
@@ -0,0 +1,11 @@
+{
+	"commentStamp" : "",
+	"super" : "ClapCommand",
+	"category" : "Clap-Core-Main-Exemple",
```

Comment:
there should probably be a `Clap-Examples` package besides `Clap-Core` and `Clap-Tests`